config/initializers/dqt.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "dqt"
+
+Dqt.configure do |config|
+  def json_to_hash(json)
+    JSON.parse(json.gsub("=>", ":"), symbolize_names: true)
+  end
```

Comment:
If we don't store the params in an environment, we don't need this method, I think.

Reply:
We do in dev